src/App.js

```diff
@@ -27,9 +36,25 @@ export default class App extends React.Component {
         }
         return (
             <div className="content-container">
-                <Search />
-                <MovieDetails movie={ movies.data[0] } />
-                <MovieList movies={ movies.data } />
+                
+                
+                <Router> 
+                    <div>
+                        <Search />
+                        <ul>
+                            <li><Link to='/'>home</Link></li>
+                            <li><Link to='/search'>search</Link></li>
+                            <li><Link to='/details'>details</Link></li>
+                        </ul>
+                        <Switch>
+                            <Route path='/' exact render={ (props) => <MovieList {...props} movies={ movies.data } /> } />
+                            <Route path='/search' exact render={ (props) => <MovieList {...props} movies={ movies.data } /> } />
+                            <Route path='/search/:searchString' render={ (props) => <MovieList {...props} movies={ movies.data } /> } />
+                            <Route path='/details/:id' render={ (props) => <MovieDetails {...props} movies={ movies.data }/> } />
```

Comment:
I would replace these arrow functions with class methods that are bound to this context.